dll/win32/shell32/utils.cpp

```diff
@@ -204,6 +204,59 @@ HRESULT SHILAppend(_Inout_ LPITEMIDLIST pidl, _Inout_ LPITEMIDLIST *ppidl)
     return hr;
 }
 
+/*************************************************************************
+ *  InternalExtractIconListW [SHELL32.238]
```

Comment:
This function is exported by name, not by ordinal (according to Geoff and my Win10 has a different function @ 238).

Reply:
```suggestion
 *  InternalExtractIconListW [SHELL32.@]
```
To complete what @whindsaks said:
- There exists both InternalExtractIconListA and InternalExtractIconListW _exported by name_, on Windows 2003, Windows 7 (and perhaps Win10 as well);
- InternalExtractIconListA just returns NULL without doing anything;
- On Windows 2003, InternalExtractIconListA is at ordinal 229 and the W version at ordinal 238.
- On Windows 7, InternalExtractIconListA is at ordinal 300 and the W version at ordinal 301.
- On Windows 7, shell32 ordinal 238 is CommandLineToArgvW.
- On Windows 7, _**BOTH**_ InternalExtractIconListA _**AND**_ InternalExtractIconListW are stubs that only return NULL.

Reply:
If they just return NULL then they probably are deprecated stubs now. Do we know any apps that actually need this function or is the PR pointless?

Reply:
Vista+ implementation: this is current stub behavior, that code can be moved to implementation file.
Vista+ test: nothing wrong with adding it.
S2003- implementation and test: to be decided, for each.